CDCgov/prime-reportstream
# 7580 Bundle FHIR Updates

This PR updates the HL7-FHIR mapping for MSH to Bundle based on spec updates.

The relevant spec is available here: [HL7 Segment - FHIR R4: MSH[Bundle]](https://docs.google.com/spreadsheets/d/1oOy6JW4Mv5VL8DsoA05bYZCUa1JqP4GCYNPF5GJTLxQ/edit#gid=0)

Test Steps:
1. Build this branch and run the unit and integration tests. Verify they all pass. Alternatively you can verify the Github Actions build passes. Note the HL7 to FHIR to HL7 integration test data was not changed, so the result of our HL7 to FHIR to HL7 conversion remains the same = no data changes.
2. Optional: Manually translate a HL7 document, and at the top of the resulting FHIR file, check that the Bundle has an `identifier/system` value and that it matches the text in the PR changes.

## Changes
- Added new `system` value to the identifier for Bundles
- Updated a data sample for reference

## Linked Issues
- #7580
